cypress/e2e/home_page_spec.cy.ts

```diff
@@ -0,0 +1,5 @@
+describe('Home page', () => {
+	it('Successfully load home page', () => {
+		cy.visit('http://localhost:3000/');
```

Comment:
I would like to just see this test called `smoke.cy.ts` and checks if it can see element which is displayed on first page. Currently, our app could be broken but still be able to navigate to this URL. And also since you have set `baseUrl`, you could just pass cy.visit("/"), but we dont want to to this.

Reply:
It's called smoke.cy.ts and it checks if there is a h1 displayed on first page.